### PR TITLE
solving the add two numbers problem (day 11, second point).

### PR DESCRIPTION
**Pull request by CHAriina**

### CODE REVIEW

File: ScnSoft-Education/LeetCodeTraining.Tests/AddTwoNumbers.Tests.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
+﻿using LeetCodeTraining.AddTwoNumbers9;
+namespace AddTwoNumbersTests
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining.Tests/AddTwoNumbers.Tests.cs]: fix namespace name

File: ScnSoft-Education/LeetCodeTraining/AddTwoNumbers9/AddTwoNumbers.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
+﻿namespace LeetCodeTraining.AddTwoNumbers9
+{
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining/AddTwoNumbers9/AddTwoNumbers.cs:2]: apply 'format document'

File: ScnSoft-Education/LeetCodeTraining.Tests/AddTwoNumbers.Tests.cs (hunk 1 above)
Comment (CHAriina) [reply ScnSoft-Education/LeetCodeTraining.Tests/AddTwoNumbers.Tests.cs]: fixed the namespace

File: ScnSoft-Education/LeetCodeTraining.Tests/AddTwoNumbers.Tests.cs [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                Assert.Equal(expectedList.val, result.val);
+                expectedList = expectedList.next;
+                result = result.next;
+            }
+        }
+
+        private ListNode CreateLinkedList(int[] values)
+        {
+            if (values == null || values.Length == 0)
+                return null;
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining.Tests/AddTwoNumbers.Tests.cs:32]: use one code style format. In all other cases you have {... } after if construction

File: ScnSoft-Education/LeetCodeTraining/AddTwoNumbers9/AddTwoNumbers.cs [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+            {
+                int value1 = (currentNodeL1 != null) ? currentNodeL1.val : 0;
+                int value2 = (currentNodeL2 != null) ? currentNodeL2.val : 0;
+                int sum = carry + value1 + value2;
+                carry = sum / 10;
+
+                resultNode.next = new ListNode(sum % 10);
+                resultNode = resultNode.next;
+
+                if (currentNodeL1 != null) currentNodeL1 = currentNodeL1.next;
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining/AddTwoNumbers9/AddTwoNumbers.cs]: use one code style format. In all other cases you have {... } after if construction

File: ScnSoft-Education/LeetCodeTraining/AddTwoNumbers9/AddTwoNumbers.cs (hunk 2 above)
Comment (CHAriina) [reply ScnSoft-Education/LeetCodeTraining/AddTwoNumbers9/AddTwoNumbers.cs:2]: added "document formatting"

File: ScnSoft-Education/LeetCodeTraining.Tests/AddTwoNumbers.Tests.cs (hunk 3 above)
Comment (CHAriina) [reply ScnSoft-Education/LeetCodeTraining.Tests/AddTwoNumbers.Tests.cs:32]: one code style format is used.

File: ScnSoft-Education/LeetCodeTraining/AddTwoNumbers9/AddTwoNumbers.cs (hunk 4 above)
Comment (CHAriina) [reply ScnSoft-Education/LeetCodeTraining/AddTwoNumbers9/AddTwoNumbers.cs]: one code style format is used.

**Pull request merged**